_layouts/home.html

```diff
@@ -46,5 +54,43 @@ <h2>SCHEDULE</h2>
 </section>
 <br><br>
 
+<section id="Interest Form" class="index-schedule">
+    <div class="container schedule-body">
+        <div class="row">
+            <section class="col-sm-12">
+                <h2>For More information please fill out this form</h2>
+                <iframe src="https://docs.google.com/forms/d/e/1FAIpQLSe4G1Dl1mrnlTnKrIXhlsAmsvUzTmCNywHFqcVDVnXx3fH4Pg/viewform?embedded=true" width="640" height="1547" frameborder="0" marginheight="0" marginwidth="0">Loading…</iframe>
```

Comment:
I don't much like the styles on the iframe. It is not mobile friendly at all.

Reply:
I think I have fixed it. I put the items in containers in css form.